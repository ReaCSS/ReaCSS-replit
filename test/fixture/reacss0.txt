#input
{ HaHa; }
{ TestRule }
{}
{
  This one has more than one declarations;
  See?;
  There is three!
}
#output

#other
{}
{}
{}
{}
{many;}
{many; many}
{really}
{too}
{many; decls;}
{Could; you; handle; them; all; including; this; super; duper; crazy; long; decl; you; have; never; seen; before; and; will; never; after; and; and; and; and; and; and; ever; }
{I; do; not; think; so; LoL}
{}
{}
#another