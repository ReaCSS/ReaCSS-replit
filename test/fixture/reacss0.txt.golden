#input {   decl; decl } {   decl } {   decl } {   decl; decl; decl } #output